Backpack/src/main/res/values/backpack.color.xml

```diff
@@ -18,126 +18,126 @@
 -->
 <resources>
 
-	<color name="bpkWhite">#ffffff</color>
+	<color name="bpkBlue100">#cbeef5ff</color>
```

Comment:
Doesn't android use AARRGGBB so this is actually wrong. Should be `ffcbeef5` I think

Reply:
I think you're right, but we're getting that value from the tokens, so we either change the tokens or do some string manipulation in the gulp file, both not ideal 🤔

Reply:
Yeah it's quite strange that we are putting these values in `raw.android.json` in the first place. cc @matthewdavidson

Reply:
I think [this](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/gulpfile.babel.js) is the issue. We should use the `hex8argb` transform